private/buf/cmd/buf/command/registry/registrylogin/registrylogin.go

```diff
@@ -140,31 +157,33 @@ func inner(
 			return err
 		}
 	}
-	// Do not print unless we are prompting
-	if !flags.TokenStdin {
-		if _, err := fmt.Fprintf(
-			container.Stdout(),
-			"Enter the BSR token created at https://%s/settings/user.\n\n",
-			remote,
-		); err != nil {
-			return err
-		}
+	if flags.TokenStdin && flags.NoBrowser {
+		return fmt.Errorf("cannot use both --%s and --%s flags", tokenStdinFlagName, noBrowserFlagName)
```

Comment:
This doesn't make sense to me - wouldn't you *exclusively* want to use `--token-stdin` when you are *not* using the browser?

I think this makes more sense if you name the flag for what it will do, not what it won't - ie `--prompt`

Reply:
Is there a best practise for mentioning the flag? `--%s` or `%q`?

Reply:
Changing to `--%s`.